easybuild/easyconfigs/a/assembly-stats/assembly-stats-1.0.1-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'CMakeMake'
+
+name = 'assembly-stats'
+version = '1.0.1'
+
+homepage = 'https://github.com/sanger-pathogens/assembly-stats'
+description = 'Get assembly statistics from FASTA and FASTQ files.'
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+source_urls = ['https://github.com/sanger-pathogens/assembly-stats/releases']
+sources = [{
+    'git_config': {
+        'url': 'https://github.com/sanger-pathogens',
+        'repo_name': name,
+        'tag': 'v%(version)s',
+        'recursive': True,
+    },
+    'filename': SOURCE_TAR_GZ,
+}]
+patches = ['%(name)s-1.0.1_fix_str_cast.patch']
+checksums = [
+    # assembly-stats-1.0.1.tar.gz 
+    'ab8fd8a3ab79b9141d09b9f46d86aa8458ec244b568fa1e55c6eccc210f4984e',
+    # assembly-stats-1.0.1_fix_str_cast.patch 
+    'e17ab66e72d5355c9f84b06da1133ca8370c15a195b77c515b9a1ee5de888c5a',
+]
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('CMake', '3.24.3'),
+    ('pkgconf', '1.8.0'),
+]
+
+dependencies = [
+    ('zlib', '1.2.12'),
+]
+
+configopts = '-DINSTALL_DIR=%(installdir)s/bin'
+
+runtest = 'test'
+
+sanity_check_paths = {
+    'files': ['bin/assembly-stats'],
+    'dirs': []
+}
+
```

Comment:
if possible, please also try adding a `sanity_check_commands = ['assembly-stats --help']` or something like that (if it exists and returns 0)